components/SearchForm.vue

```diff
@@ -1,8 +1,8 @@
 <template>
 	<div class="search-container">
-		<form id="search-form" action="">
+		<form id="search-form" action="/search">
 			<label for="search">Search for:</label>
-			<input id="search" type="search" placeholder="Search..." name="search">
+			<input id="search" type="search" placeholder="Search..." name="s">
```

Comment:
why `s` instead of `search`?

Reply:
Because the search page uses the `s` URL parameter to determine the search query: https://github.com/inclusive-design/wecount.inclusivedesign.ca/blob/master/pages/search/_query/index.vue#L23